README.md

```diff
@@ -130,14 +118,63 @@ By default, this will create a set of certificates in `/etc/grafana` that are ba
 
 _Note:_ Enabling SSL on Grafana will not allow for running on privileged ports such as `443`. To enable this capability you can use the suggestions documented in [this Grafana documentation](http://docs.grafana.org/installation/configuration/#http-port)
 
+### Allow access to PE-managed postgres nodes with the following class:
+
+This is required for collection of postgres metrics.  The class should be applied to the master (or postgres server if using external postgres).
+
+```
+class { 'puppet_metrics_dashboard::profile::master::postgres_access':
+  telegraf_host => 'grafana-server.example.com',
+}
+```
+
+`telegraf_host` is optional.  If you do not specify it, the class will look for a node with the `puppet_metrics_dashboard` class applied in PuppetDB and use the `certname` of the first host returned.  If the PuppetDB lookup fails and you do not specify `telegraf_host` then the class outputs a warning.
```

Comment:
Should this not be using a local Telegraf agent instead of `'grafana-server.example.com'`?